### DISCUSSION TITLE
Fullscreen games

### DISCUSSION THREAD

**Discussion by djneba** [lingtjien/Grid-Tiling-Kwin#143] [Q&A]
How can I run games fullscreen? I tried to blacklist them, but they still get tiled. Assume I know nothing.

**Comment by lingtjien** [comment]
blacklist them, but I assume you don't have the correct window name if it doesn't work. What is the config value for blacklist that you use? And what is the window class (application)? You can find this using the `Windows Operations Menu` global shortcut as specified in the README.md

**Comment by djneba** [comment]
Ok. So what would I need to write in the blacklist to include all .exe windows? Or do I need to add them one by one? How to separate items in the blacklist?

**Comment by lingtjien** [comment]
As the readme specifies, matching is done based on regex. There are many tools out there that you can use to build regex strings and test them. (https://regexr.com/ is one I use often myself)

If a window has a class name that ends with `.exe` you could use the following `\.exe$` (if you put that in https://regexr.com/ expression you will get an explanation of what all the tokens do)

If you want to match only specific names you could do `some-app|another-app|another-one` (partial match)

**Comment by djneba** [comment]
Ok. Great. It works like a charm. Thank you.
